integration-tests/test.Dockerfile

```diff
@@ -2,16 +2,26 @@ ARG BASE_IMAGE
 ARG IMAGE_VERSION=latest
 FROM ${BASE_IMAGE}:${IMAGE_VERSION} AS build-env
 
-ARG SUITES=chaos migration performance reorg smoke soak benchmark
-
-COPY . testdir/
 WORKDIR /go/testdir
+RUN mkdir -p /go/testdir/integration-tests/load
+COPY go.mod go.sum  ./
+COPY integration-tests/go.mod integration-tests/go.sum ./integration-tests/
+COPY integration-tests/load/go.mod integration-tests/load/go.sum ./integration-tests/load/
+RUN cd integration-tests && go mod download
+RUN cd integration-tests/load && go mod download
```

Comment:
@anirudhwarrier how about `integration-tests/ccip-tests/`. Do we need to include it here?

Reply:
@lukaszcl `integration-tests/ccip-tests/` is not a separate go module and therefore no go.mod file there.